unime/src/hooks.server.ts

```diff
@@ -1,27 +1,18 @@
 import type { Handle, HandleServerError } from '@sveltejs/kit';
 
 export const handle = (async ({ event, resolve }) => {
-  // info(`hooks.server.ts: event.url: "${event.url}"`);
-  console.log(`hooks.server.ts: event.url: "${event.url}"`);
-
   const response = await resolve(event);
-
-  // info(`hooks.server.ts: response.url: "${response.url}"`);
   return response;
 }) satisfies Handle;
 
-export const handleError: HandleServerError = async ({ error, event, status, message }) => {
+export const handleError: HandleServerError = async ({ error }) => {
   const errorId = crypto.randomUUID();
 
-  // example integration with https://sentry.io/
-  // Sentry.captureException(error, {
-  //   extra: { event, errorId, status },
-  // });
-
+  // eslint-disable-next-line no-console
```

Comment:
this comment is not very clarifying for me

Reply:
It's an instruction to ESLint. Turn off rule "no-console" in next line only to mute the warning.